conda/testing/helpers.py

```diff
@@ -311,129 +321,76 @@ def _patch_for_local_exports(name, subdir_data, channel, index):
     subdir_data._mtime = float("inf")
 
 
-# this fixture appears to introduce a test-order dependency if cached
-def get_index_r_1(subdir=context.subdir):
-    with open(join(TEST_DATA_DIR, "index.json")) as fi:
-        packages = json.load(fi)
+def _get_index_r_base(
+    json_filename,
+    channel_name,
+    subdir=context.subdir,
+    add_pip=False,
+):
+    with open(join(TEST_DATA_DIR, json_filename)) as fi:
+        all_packages = json.load(fi)
```

Comment:
come now, we're not writing math here

```suggestion
    with open(join(TEST_DATA_DIR, json_filename)) as test_file:
        all_packages = json.load(test_file)
```

Reply:
For the record I didn't write that block, only moved it around 😬 There's plenty of "math-like variable names" in this module and I tried to keep the diff as short as reasonably possible.

Reply:
ah, d'oh, you're right :)